Source/Blazorise/Components/NumericEdit/NumericEdit.razor.cs

```diff
@@ -221,6 +221,16 @@ protected override string GetFormatedValueExpression()
     /// </summary>
     protected string MaxString => Max.ToCultureInvariantString();
 
+    /// <summary>
+    /// Gets the min value if defined, otherwise null.
+    /// </summary>
+    protected object GetMin => minDefined ? Min : null;
```

Comment:
Hm, naming is not right. It has a Get prefix but it is properly, not a method. Either we make it a method. Or rename it.

Proposals

- SafeMin/SafeMax
- MinValue/MaxValue
- MinAttribute/MaxAttribute
- MinAttributeValue/MaxAttributeValue

Reply:
Done.